Week1/homework/js-exercises/helloworld.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+//printing statement
+console.log('Hello world!'); //English
+console.log('Hallo Wereld!'); //Dutch
+console.log('Halo, dunia!'); //Indonesian
+console.log('Ciao, mondo!'); //Italian
+console.log('Hola, mundo!'); //Spanish
+console.log('Bonjour le monde'); //France
+console.log('Привет, мир'); //russian
+console.log('こんにちは世界'); //Japanese
+console.log('مرحبا'); //Arabic
+console.log('नमस्ते दुनिया'); //Hindi
```

Comment:
Seems to be working just fine! Well done :)

Reply:
this part was east as bit of cake